apps/src/turtle/artist.js

```diff
@@ -1344,7 +1344,8 @@ Artist.prototype.displayFeedback_ = function() {
     appStrings: {
       reinfFeedbackMsg: turtleMsg.reinfFeedbackMsg(),
       sharingText: turtleMsg.shareDrawing()
-    }
+    },
+    enablePrinting: true
```

Comment:
Philosophical question for the future: should this configuration be set on the back-end or the front-end?

Reply:
this is really interesting... i don't have a clear answer but i do have some thoughts:
- the frontend renders/controls the visualization, which is what ends up being printed, _but_
- the backend stores/serves the source code, which also determines the content of visualization

this setting is ultimately a lab-level setting -- we expect XLab to have a visualization that is printable (or even worth printing). this makes me think it should be a backend setting.

however, if we were to change the renderer in Artist, that would happen on the frontend, and may change whether or not Artist levels should be printable. does that make it a frontend setting???

i'm definitely thinking too hard about this 😅 but i thought it was an interesting question

Reply:
🤔  I'm leaning toward backend, but it's not 100% clear cut to me. An Artist level is responsible for knowing whether it's printable or not and then based on that the frontend is responsible for hide/show the print button.

Reply:
yeah, i think that's a reasonable choice given our current assumption that a level/lab type should either be printable or not